modules/hosted-restart-hcp-components.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// * hosted_control_planes/index.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-restart-hcp-components_{context}"]
+= Restarting hosted control plane components
+
+If you are an administrator for hosted control planes, you can use the `hypershift.openshift.io/restart-date` annotation to restart all control plane components for a particular `HostedCluster` resource. For example, you might need to restart control plane components for certificate rotation.
+
+.Procedure
+
+To restart a control plane, annotate the `HostedCluster` resource by entering the following command:
+
+[source,terminal]
+----
+$ oc annotate hostedcluster -n clusters example hypershift.openshift.io/restart-date=$(date --iso-8601=seconds)
+----
+
+//lahinson - june 2024 - are any of the values in the above command user-replaced variables? If so, we need to format them in angle brackets and provide instructions for what the user should replace them with.
+
+The control plane is restarted whenever the value of the anonotation changes. The `date` command in the example serves as the source of a unique string. The annotation is treated as a string, not a timestamp.
+
+The following components are restarted:
+
+* catalog-operator
+* certified-operators-catalog
+* cluster-api
+* cluster-autoscaler
+* cluster-policy-controller
+* cluster-version-operator
+* community-operators-catalog
+* control-plane-operator
+* hosted-cluster-config-operator
+* ignition-server
+* ingress-operator
+* konnectivity-agent
+* konnectivity-server
+* kube-apiserver
+* kube-controller-manager
+* kube-scheduler
+* machine-approver
+* oauth-openshift
+* olm-operator
+* openshift-apiserver
+* openshift-controller-manager
+* openshift-oauth-apiserver
+* packageserver
+* redhat-marketplace-catalog
+* redhat-operators-catalog
+
+
+//.Verification
```

Comment:
Placeholders for future additions, I assume.

Reply:
Yep. Removed.